build.gradle

```diff
@@ -64,16 +64,16 @@ task integrationTest(type: Test) {
 }
 
 dependencies {
-    compile 'io.fabric8:openshift-client:5.0.2'
-    compile 'io.fabric8:servicecatalog-client:5.0.2'
+    compile 'io.fabric8:openshift-client:5.7.2'
+    compile 'io.fabric8:servicecatalog-client:5.7.2'
     compile 'org.apache.commons:commons-compress:1.19'
     compile 'org.apache.commons:commons-exec:1.3'
     compile 'org.apache.commons:commons-lang3:3.9'
-    compile 'com.redhat.devtools.intellij:intellij-common:1.2.1'
+    compile 'com.redhat.devtools.intellij:intellij-common:1.3.0-SNAPSHOT'
```

Comment:
shouldn't be 1.3.0 as it's now released ?

Reply:
updated.